slidechain/export.go

```diff
@@ -90,14 +98,27 @@ func (c *Custodian) pegOutFromExports(ctx context.Context) {
 			}
 
 			log.Printf("pegging out export %x: %d of %s to %s", txid, amounts[i], asset.String(), exporters[i])
-			// TODO(vniu): flag txs that fail with unretriable errors in the db
+
+			peggedOut := pegOutOK
```

Comment:
I think we can get rid of the `pegOutState` enum and just have a bool for retrying now! since that is the only condition we check